src/main/java/io/github/jsonSnapshot/SnapshotMatcher.java

```diff
@@ -32,10 +32,14 @@
     private static List<Snapshot> calledSnapshots = new ArrayList<>();
 
     public static void start() {
```

Comment:
Maybe you should overload "start" method to provide a custom Config that implements SnapshotConfig interface

Reply:
Following previous suggestion of SnapshotConfig interface!